recipes/xrandr.lua

```diff
@@ -0,0 +1,146 @@
+--- Separating Multiple Monitor functions as a separeted module (taken from awesome wiki)
```

Comment:
As we both failed to find the real author of this script, should it stay 'authorless' ?